modules/system/activation-scripts.nix

```diff
@@ -113,8 +113,8 @@ in
 
       ${cfg.activationScripts.preActivation.text}
 
-      ${cfg.activationScripts.createRun.text}
       ${cfg.activationScripts.checks.text}
+      ${cfg.activationScripts.createRun.text}
```

Comment:
This is also a potential point where out-of-tree checks could break, if people assume that `/run` will already exist. I think this is still a worthwhile change to make, but worth pointing out.

Reply:
I think this is definitely an improvement that we should make, just noting the potential issue here in case we see it in the future.